_headers

```diff
@@ -1,2 +1,4 @@
 /search.json
   Access-Control-Allow-Origin: *
+/search/index.js
```

Comment:
We want to be able to access this from a different domain that support.dnsimple.com.